mappings/net/minecraft/entity/EntityWithAiGoals.mapping

```diff
@@ -1,4 +1,17 @@
 CLASS us net/minecraft/entity/EntityWithAiGoals
-	FIELD a pos Ldx;
+	FIELD a aiCenter Ldx;
+	FIELD b aiRadius F
 	FIELD bv ID Ljava/util/UUID;
+	METHOD <init> (Lali;)V
+		ARG 0 world
+	METHOD a setAiRange (Ldx;I)V
+		ARG 0 center
+		ARG 1 radius
 	METHOD cO canSpawn ()Z
+	METHOD dd isInAiRange ()Z
+	METHOD de getAiCenter ()Ldx;
+	METHOD df getAiRadius ()F
```

Comment:
Similar to `getAiHome`, I want to keep `Ai` in the name. I think this existing name is good.
I reviewed the scuttlebug transportation section of the 0.5 A Press video and "home" is always a single point. The area around the home is separate.